src/quo2/components/selectors/react/style.cljs

```diff
@@ -0,0 +1,19 @@
+(ns quo2.components.selectors.react.style)
+
+(def container
+  {:flex-direction  :row
+   :justify-content :flex-start
+   :flex            1
+   :flex-wrap       :wrap
+   ;; FIXME: after upgrading to >rn71.0, use :row-gap to
```

Comment:
Personally I would't leave this comment here. I think we have this issue all over the codebase until we upgrade. Perhaps better to document this in a generic issue of where we can eventually make these refactors? 🤔

Reply:
Is there one already? If not, I'll make an issue to scan the FIXME/NOTEs across the app and collect them in another issue.

Reply:
I don't think I've seen any other fix me's etc related to this in the codebase.

Perhaps a more general, things we can do after upgrade exists? Maybe @siddarthkay knows if such a doc exists?

Reply:
Added [here](https://www.notion.so/Post-RN-upgrade-changes-e0ab9a017b9547b6ae8cde6aa7ef060d)

Reply:
@clauxx are you intending to remove or keep these comments? or the pr needs to be updated? 🤔